dcpy/test/connectors/edm/test_publishing.py

```diff
@@ -112,6 +114,90 @@ def test_legacy_upload(
         assert not s3.exists(TEST_BUCKET_NAME, f"{TEST_PRODUCT_NAME}/{prefix}/{path}")
 
 
+@patch("dcpy.connectors.edm.publishing.BUILD_NAME", TEST_BUILD)
```

Comment:
interesting use of `patch`, this effectively patches an envar?

Reply:
no, this line here overrides `BUILD_NAME` variable in `publishing.py`. I tried to do an envar but it wasn't working

Reply:
There's some weirdness with env vars and reloading the configuration module - I think this approach makes sense, though also happy to chat about in which cases reloading the configuration module works and in which cases it doesn't actually reload the env var